products/quarkus.md

```diff
@@ -43,6 +43,7 @@ releases:
     lts: true
     releaseDate: 2024-09-25
     eol: 2025-09-25
+    eoes: false
```

Comment:
why do we need this?

Reply:
https://endoflife.date/quarkus-framework --> "RHBQ" column

Reply:
Also, see https://github.com/endoflife-date/endoflife.date/blob/master/products/quarkus.md?plain=1#L14

Reply:
then sounds like we should have this for every columns

Reply:
We do. Each Quarkus LTS release is being made available (and maintained) by Red Hat as a RHBQ release. Hence, each LTS release with the exception of 2.13, which was before they established the LTS program, also has the `eoes` flag set to `false`.

Reply:
can you also add a note before L34 for future ref? Thanks!

Reply:
Done.